src/settings.cpp

```diff
@@ -557,6 +560,19 @@ void read_settings_xml(pugi::xml_node root)
     }
     model::external_sources.push_back(make_unique<FileSource>(path));
   }
+  std::vector <double> source_strength ;
+  for (auto& s : model::external_sources)
+  {
+    source_strength.push_back(s->strength()); //is that strength normalized? if yes then it can be used as a pdf 
+  }
+  */
+  //external_sources_alias_sampler = DiscreteIndex(model::external_sources);
+  //external_sources_alias_sampler.init_alias();
+  vector<double> strengths;
+  for (auto& s : model::external_sources) {
+    strengths.push_back(s->strength());
+  }
```

Comment:
These lines look redundant with the ones above - perhaps the ones above can be removed?